test/hotspot/jtreg/runtime/ErrorHandling/VeryEarlyAssertTest.java

```diff
@@ -49,8 +49,7 @@ public class VeryEarlyAssertTest {
   public static void main(String[] args) throws Exception {
 
 
-    ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
-            "-version");
+    ProcessBuilder pb = ProcessTools.createTestJvm("-version");
```

Comment:
I don't think this is necessary. IIUC this test triggers an assertion failure during the static initialization that occurs in debug.o which will happen when libjvm is loaded, well before any argument processing could occur, or have any impact.

Reply:
David, thank you for providing the details. I agree, and have updated the test.